src/main/java/frc/robot/subsystems/swerve/SwerveSubsystem.java

```diff
@@ -794,13 +792,11 @@ public Rotation2d getRotation() {
 
   /** Sets the current odometry pose. */
   public void setPose(Pose2d pose) {
-    this.pose = pose;
     try {
       estimator.resetPosition(lastGyroRotation, getModulePositions(), pose);
     } catch (Exception e) {
       System.out.println("odo reset sad :(");
```

Comment:
Should anything else happen here? Should we at least log the exception and know *why* it's sad?